src/include/ipxe/in.h

```diff
@@ -23,6 +23,7 @@ FILE_LICENCE ( GPL2_OR_LATER_OR_UBDL );
 #define	INADDR_NET_CLASSA	htonl ( 0xff000000 )
 #define	INADDR_NET_CLASSB	htonl ( 0xffff0000 )
 #define	INADDR_NET_CLASSC	htonl ( 0xffffff00 )
+#define	INADDR_NET_RFC3021	htonl ( 0xfffffffe )
```

Comment:
Please change to `INADDR_NET_P2P` or some other similar name that reflects the purpose of the macro, since RFC numbers are not meaningful to the reader without an external reference.